Lib9c.DevExtensions/Action/Stage/ClearStage.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Action;
+using Nekoyume.Model;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+
+namespace Lib9c.DevExtensions.Action.Stage
+{
+    [Serializable]
+    [ActionType("clear_stage")]
+    public class ClearStage : GameAction
+    {
+        public Address AvatarAddress { get; set; }
+        public int TargetStage { get; set; }
+
+        public override IAccountStateDelta Execute(IActionContext context)
+        {
+            if (context.Rehearsal)
+            {
+                return context.PreviousStates;
+            }
+
+            var states = context.PreviousStates;
+            var worldInformation = new WorldInformation(
+                context.BlockIndex,
+                states.GetSheet<WorldSheet>(),
+                TargetStage
+            );
+            return states.SetState(AvatarAddress.Derive(SerializeKeys.LegacyWorldInformationKey),
+                worldInformation.Serialize()
+            );
```

Comment:
It's enough to clear the stage.👍 One thing in the real scenario, Avatar's quest list will be updated with the inventory. But that's OK, because the actions in DevEx are not as strict as in reality.🙂